### DISCUSSION TITLE
Field 模型中的 `path` 与 `address` 有什么区别，作用是什么？

### DISCUSSION THREAD

**Discussion by WangJincheng4869** [alibaba/formily#4103] [Q&A 2.x]
在文档中仅说明了是什么东西，但是不知道具体作用。

![图片](https://github.com/alibaba/formily/assets/41162158/6d1e072a-c6c9-40a5-8f4d-84c4f11bb6eb)

如果我想使用 `Form.setFieldState` 方法改变字段的状态，我应该使用 `path` 还是 `address` ？

**Comment by jiayongchang** [comment]
<img width="1069" alt="image" src="https://github.com/alibaba/formily/assets/67669980/4c4eb840-f155-46f2-8fb2-7408072cafc2">
没想到吧，写在这了

**Comment by franckXu** [comment]
我理解，一个是值模型的路径，一个字段模型的路径
